env/includes/unreal/tk-unreal-location.yml

```diff
@@ -6,6 +6,7 @@
 
 tk-unreal.location:
   version: v1.2.3
-  type: git
-  path: git@github.com:GPLgithub/tk-unreal.git
-#  path: git@github.com:ue4plugins/tk-unreal.git
+  type: github_release
+  organization: GPLgithub
+#  organization: ue4plugins
```

Comment:
Should the comment be flipped here? Should the GPL org remain? Not sure of the history/intent, so just checking.

Reply:
I removed GPLgithub. But that means that a release will have to be created on ue4plugins.